README.md

```diff
@@ -53,6 +53,14 @@ Currently, we provide wheels for: Windows x86-64, macOS x86-64, macOS arm64, and
 Linux x86-64. The Linux wheels are built for manylinux_2_28, which makes them
 broadly compatible (Debian 10, Ubuntu 18.10, Fedora 29).
 
+On Linux and macOS, CMake's `find_package` command should find Halide as long as
```

Comment:
Maybe we need to prefix this with "For C++ usage of the pip package:" or similar.

Reply:
Done!